src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js

```diff
@@ -115,8 +115,7 @@ class VulnerabilitiesCveTableToolbar extends Component {
                     <GridItem span={ 1 } />
                     <GridItem span={ 2 }>
                         <DownloadButton onSelect={ downloadReport } />
-                        <div className="number-of-results">{ totalNumber }</div>
-                        Results
+                        <div className="number-of-results">{ totalNumber } Results</div>
```

Comment:
Can we use results section from https://github.com/RedHatInsights/insights-frontend-components/blob/master/doc/components/tableToolbar.md instead, please?

Reply:
I would like but not to be sure because UI of `<TableToolbar results={ totalNumber }>` doesn't looking the same as design before or as design in new wireframe. So we need more discussion to this.

Reply:
Can you please attach screenshot how it looks like?